homeassistant/components/utility_meter/__init__.py

```diff
@@ -134,15 +137,15 @@ async def async_setup(hass, config):
             )
 
             # add one meter for each tariff
-            tariff_confs = []
+            tariff_confs = {}
             for tariff in conf[CONF_TARIFFS]:
-                tariff_confs.append(
-                    {
-                        CONF_METER: meter,
-                        CONF_NAME: f"{meter} {tariff}",
-                        CONF_TARIFF: tariff,
-                    }
-                )
+                name = f"{meter} {tariff}"
+                tariff_confs[name] = {
+                    CONF_METER: meter,
+                    CONF_NAME: f"{meter} {tariff}",
```

Comment:
why not using `name` here ?

Reply:
Good spot - I had planned to but was probably juggling too many PRs at once.

Reply:
Do you still approve after my changes?

Reply:
Yes